app/uk/gov/hmrc/eoricommoncomponent/frontend/views/subscription/sub01_outcome_processing.scala.html

```diff
@@ -18,33 +18,25 @@
 @import uk.gov.hmrc.eoricommoncomponent.frontend.views.ServiceName._
 @import uk.gov.hmrc.eoricommoncomponent.frontend.views.helpers.DateFormatter
 @import uk.gov.hmrc.govukfrontend.views.html.components._
+@import uk.gov.hmrc.eoricommoncomponent.frontend.viewModels.Sub01OutcomeViewModel.headingForProcessing
 
-@this(layout_di: layout, feedback_di: helpers.feedback, dateFormatter: DateFormatter, govukPanel : GovukPanel)
+@this(layout_di: layout, feedback_di: helpers.feedback, dateFormatter: DateFormatter, govukPanel: GovukPanel, h2: helpers.h2, p: helpers.paragraph)
 
 @(orgName: Option[String], processedDate: String)(implicit messages: Messages, request: Request[_])
 
-@heading = {@orgName match {
-    case Some(org) => {@messages("cds.sub01.outcome.processing.heading", shortName, org)}
-    case None => {@messages("cds.sub01.outcome.processing.heading-noname", shortName)}
-  }
-}
-
 @confirmationHtml = {
     <p id="processed-date" class="govuk-!-font-size-36"><strong>@messages("cds.subscription.outcomes.in-processing.received", dateFormatter.format(processedDate))</strong></p>
```

Comment:
We don't want to be using <strong> tags, this could be changed to something such as `@p(messages("cds.subscription.outcomes.in-processing.received", dateFormatter.format(processedDate))`